tests/PSModuleTest.Tests.ps1

```diff
@@ -1,30 +1,36 @@
-﻿Describe 'TimeSpan' {
+﻿BeforeAll {
+    . $PSScriptRoot/../src/variables/private/UnitMap.ps1
+    . $PSScriptRoot/../src/functions/private/Format-UnitValue.ps1
+    . $PSScriptRoot/../src/functions/public/Format-TimeSpan.ps1
+}
+
```

Comment:
Remove this as its included in the test framework that is used to run the tests.